pages/merchant/[username].tsx

```diff
@@ -59,9 +60,23 @@ function ReceivePayment() {
                 alt="pin icon"
                 className={styles.pin_icon}
               />
-              <button className={styles.pin_btn}>Pin to homescreen</button>
+              <button
+                onClick={() =>
+                  dispatch({
+                    type: ACTIONS.PINNED_TO_HOMESCREEN,
+                    payload: !state.pinnedToHomeScreen,
```

Comment:
```pinnedToHomeScreen``` is a confusing name to me, do you mean ```pinnedToHomeScreenModalVisible```? If so, I would adjust the name used through out the change.